thousandeyes/schemas/common_schema.go

```diff
@@ -311,748 +467,510 @@ var schemas = map[string]*schema.Schema{
+		Required:    false,
 		Type:        schema.TypeSet,
-		Description: "The array of label objects.",
-		Computed:    true,
 		Elem: &schema.Resource{
-			// Schema definition here is to support group objects returned from
-			// reads of test resources.
 			Schema: map[string]*schema.Schema{
-				"agents": {
-					// See `agents-label` rather than `agents`
-					Type:        schema.TypeList,
-					Description: "Define the ThousandEyes agents to use.",
-					Computed:    true,
-					Elem: &schema.Resource{
-						Schema: map[string]*schema.Schema{
-							"agent_id": {
-								Type:        schema.TypeInt,
-								Description: "The unique ThousandEyes agent ID.",
-								Computed:    true,
-							},
-						},
-					},
-				},
-				"builtin": {
-					Type:        schema.TypeBool,
-					Description: "Shows whether you are using built-in (true) labels or user-created (false) labels. Built-in labels are read-only.",
-					Computed:    true,
-				},
-				"group_id": {
-					Type:        schema.TypeInt,
-					Description: "The unique ID of the label.",
-					Computed:    true,
-				},
-				"name": {
+				"agent_id": {
 					Type:        schema.TypeString,
-					Description: "The name of the label.",
-					Computed:    true,
-				},
-				"tests": {
-					Type:        schema.TypeList,
-					Description: "The list of tests.",
-					Computed:    true,
-					Elem: &schema.Resource{
-						Schema: map[string]*schema.Schema{
-							"test_id": {
-								Type:        schema.TypeInt,
-								Description: "The unique ID of the test.",
-								Computed:    true,
-							},
-						},
-					},
+					Description: "The agent ID of the enterprise agent for the test.",
+					Optional:    true,
 				},
-				"type": {
-					// See `type-label` rather than `type`
+				"ip_address": {
 					Type:        schema.TypeString,
-					Description: "[tests, agents, endpoint_tests, or endpoint_agents] The type of label.",
-					Computed:    true,
+					Description: "IP address of the agent interface.",
+					Optional:    true,
 				},
 			},
 		},
 	},
-	"group_id": {
-		Type:        schema.TypeInt,
-		Description: "The unique ID of the label. For built-in labels, this number is a negative.",
-		Computed:    true,
+	// clientCertificate
+	"client_certificate": {
+		Type:        schema.TypeString,
+		Description: "String representation (containing newline characters) of the client certificate, if used.",
+		Optional:    true,
 	},
-	"headers": {
-		Type:        schema.TypeList,
-		Description: "[\"header: value\", \"header2: value\"] The array of header strings.",
-		Elem: &schema.Schema{
-			Type: schema.TypeString,
+	// contentRegex
+	"content_regex": {
+		Type:        schema.TypeString,
+		Description: "Verify content using a regular expression. This field does not require escaping.",
+		Optional:    true,
+		Required:    false,
+		Default:     ".*",
+	},
+	// customHeaders
+	"custom_headers": {
+		Description: "The custom headers.",
+		Optional:    true,
```

Comment:
```suggestion
		Optional:    true,
		Sensitive: true,
```
This is a change we introduced this week to `v2` we need to have it for `v3` as well